compiler/testData/diagnostics/tests/NumberPrefixAndSuffix.fir.kt

```diff
@@ -8,47 +8,47 @@ infix fun Any?.ddoo(a: Any) {}
 operator fun Any?.contains(a: Any): Boolean = true
 
 fun test(a: Any) {
-    1f<!UNRESOLVED_REFERENCE!>oo<!> a
+    <!UNRESOLVED_REFERENCE!>1foo a<!>
```

Comment:
This looks irony/funny. Now it's unclear whether the unresolved reference is `foo` or `oo`. 😀

Reply:
OK, reverted to roughly the previous behavior for all "UNRESOLVED_REFERENCE"